### DISCUSSION TITLE
Start maximized?

### DISCUSSION THREAD

**Discussion by Barugon** [FreeTubeApp/FreeTube#5790] [Q&A]
Is there a way to have the window start maximized (Fedora 40)?

**Comment by absidue** [comment]
FreeTube remembers the maximized state of the last FreeTube window that you close and then sets that on the first window you open next time you start FreeTube. So all you need to do is make sure that the last window you close is maximized.

**Comment by Barugon** [comment]
It doesn't. Every time I start it,  I have to maximize it. This is on GNOME BTW.

**Reply by absidue** [reply]
It should do, it works correctly on Windows and the code in FreeTube is the same. So if it doesn't it sounds like another Linux window manager problem (similar to how Linux window managers seems to maintain their own fullscreen flag that they don't tell the app about, so if you exit it through the window manager the app still thinks it is in fullscreen).

**Comment by Barugon** [comment]
Still an issue in v0.23.2 Beta.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Can you maybe create a screen recording of this behavior. 

1. Launch app
2. Maximize if not maximized 
3. Close app
4. Launch app

**Reply by Barugon** [reply]
https://youtu.be/BFX_qImc78A

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
workaround:

1. Launch FT
2. Press f11
3. Close FT (FT is now in fullscreen so you have to use a keyboard shortcut to quit FT, ctrl+q/w should work)
4. Launch FT
5. Press f11
6. Window should be in maximized state if not press maximize button
7. Close FT
8. Launch FT, should launch maximized everytime now until you change the window size yourself
